cypress/utils/utils.ts

```diff
@@ -363,17 +363,17 @@ export function applySearchFilter(
         ) {
             // ul[role=listbox] > li is for the Application Inventory page.
```

Comment:
While we are at it , Can these comments be removed ? Are they relevant ?

Reply:
Yes, they are still relevant.